static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -54,11 +54,11 @@ import './PublishPreview.scss'
  * )
  */
 const PublishPreviewHeader = () => {
-  const { conceptId } = useParams()
+  const { conceptId, type } = useParams()
 
-  const navigate = useNavigate()
+  const formattedType = capitalize(trimEnd(type, 's'))
```

Comment:
I'm open to doing it this way but, would it not be easier to just add `import conceptIdTypes from   if (prefix === 'V') {
    // Handle the special case for VIS prefix
    if (conceptId.startsWith('VIS')) {
      return conceptIdTypes.VIS
    }
  }
  in the `static/src/js/utils/getConceptTypeByConceptId.js` then this would not need to change any of the other components just the one place and we can test it in that util as well